extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -366,6 +366,31 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Installs the effect on the page if the there is no opt-out detected
+   * @private
+   * @return {!Promise}
+   */
+  maybeInitializeParallaxFx_() {
+    return Promise.resolve().then(() => {
```

Comment:
Can you add a comment on why this resolve (deferring to microtask) is necessary?